README.md

```diff
@@ -18,14 +18,27 @@ providers {
 }
 ```
 
-The provider takes optional configuration to specify a `kubeconfig` file:
+The provider takes couple optional configuration parameters:
```

Comment:
spelling mistake here, maybe

"takes the following optional configuration parameters"